portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -54,16 +54,8 @@
     int numberOfCommentsToDisplay = 0;
     
     @Override
-    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException { 
-      //Get number of comments.
-      numberOfCommentsToDisplay = getNumberOfCommentsToDisplay(request);
-           
-      if (numberOfCommentsToDisplay < 1 || numberOfCommentsToDisplay > 100) {
-        response.setContentType("text/html");
-        response.getWriter().println("Please enter an integer between 1 and 100.");
-        return;
-      }
-
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {      
+      System.out.println(numberOfCommentsToDisplay);
```

Comment:
Just a reminder to remove this when you're done testing.

Reply:
+1 - for the most part, avoid print statements in production code. You can use them for debugging in prod, but this is usually reserved for key steps or unexpected errors. In that case, you should include a detailed message to explain the value you are printing.